cachito/workers/pkg_managers/npm.py

```diff
@@ -24,6 +25,20 @@
 
 log = logging.getLogger(__name__)
 
+def _is_worksapce_version(version, workspaces):
+    """
+    Test if a version file path match one of the workspaces pattern string
+
+    :param version a file: version string to match with a workspace glob pattern
+    :param list workspaces: package workspaces defined in package-lock.json
+    :return: true if path matches one of the workspaces
+    :rtype: boolean
+    """
+    path = version[5:]
```

Comment:
nitpick, this might be clearer:
```suggestion
    path = version.removeprefix("file:")
```

Reply:
:+1: